src/app/policies/routes.ts

```diff
@@ -21,16 +21,6 @@ export const routes = () => {
             module: 'policies',
           },
           component: () => import('@/app/policies/views/PolicyTypeListView.vue'),
-          redirect: (to) => {
-            return {
-              ...to,
-              params: {
-                ...to.params,
-                policyPath: 'circuit-breakers',
-              },
-              name: 'policy-list-view',
-            }
-          },
```

Comment:
Because the first one could still be `CircuitBreaker` as far as I understand it? Plus why do one fix here and then a different one on `master` when the one on master is fine for both?

Reply:
Gonna resolve this one and merge (just noticed you'd approved), let me know if I got it wrong on the "potentially there are CircuitBreakers" thing.